internal/syscall/syscall_linux.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
+
```

Comment:
This should be unnecessary due to the name of the file IIUC.

Reply:
Done.